CONTRIBUTING.md

```diff
@@ -34,8 +34,8 @@ First, please make sure that [the formula belongs in this tap](#what-formulae-be
 
 ## What formulae belong here?
 
-* A new formula that works on Mac or Linux should go in Homebrew/core.
-* A new formula that is Linux-only, and a formula in Linuxbrew/core depends on it, should go in Linuxbrew/core.
-* A new formula that is Linux-only, and no formula in Linuxbrew/core depends on it, should go in Linuxbrew/extra.
+* A new formula that works on Mac or Linux should go in Homebrew/homebrew-core.
```

Comment:
I thought that Homebrew/brew is (macOS + Linux), but Homebrew/homebrew-core is still macOS-only, no?

Reply:
Good catch, thanks! Updated.

Reply:
> * A new formula that works on Mac or Linux should go in Homebrew/homebrew-core.
 
A new formula that works on both macOS and Linux should first be submitted to Homebrew/homebrew-core, then once that's merged, if it needs a fix for Linux, a separate PR can be opened for Homebrew/linuxbrew-core to add support for Linux.

Reply:
Ahhh, I thought Maxim meant specifically about "making formulae work on Linux" (ie all the `unless OS.mac?` extra dependencies).

Reply:
Fixed by https://github.com/Linuxbrew/homebrew-extra/pull/53, :crossed_fingers: